pkg/config/setup/otlp.go

```diff
@@ -72,6 +72,7 @@ func setupOTLPEnvironmentVariables(config pkgconfigmodel.Setup) {
 	// Traces settings
 	config.BindEnvAndSetDefault("otlp_config.traces.span_name_remappings", map[string]string{})
 	config.BindEnv("otlp_config.traces.span_name_as_resource_name")
+	config.BindEnv("otlp_config.traces.ignore_missing_datadog_fields")
```

Comment:
Please add a default to this new setting with the same value than in the configuration template.